### PR TITLE
Added 'neurodamus-mousify' to 'all-modules'

### PR DESCRIPTION
**Pull request by arsenius7**

### CODE REVIEW

File: bbp/modules/default.nix [hunk 1]
```diff
@@ -403,7 +403,7 @@ let
             dependencies = [ gcc neuron ];
         };
 
-        neurodamus-smousify = pkgs.envModuleGen rec {
```
Comment (arsenius7) [inline bbp/modules/default.nix]: sorry for making it more boring name

**Comment by adevress** [comment]
retriggered, the switch to Nix 2.0 had a perverted effect on Jenkins.

**Comment by adevress** [comment]
retest this please

**Comment by adevress** [comment]
retest this please

**Comment by adevress** [comment]
sorry to use your pull request as a sandbox, I needed one to prepare Jenkins for Nix 2.0 and this one is not very urgent ^^

**Comment by arsenius7** [comment]
I can live with that in exchange for having 'nix/hpc/neurodamus-mousify' module available on BG. :)

**Comment by adevress** [comment]
retest this please

**Comment by adevress** [comment]
retest this please

**Pull request merged**